src/main/java/Lesson3/Task2/Car.java

```diff
@@ -0,0 +1,16 @@
+package Lesson3.Task2;
+
+class Car extends Vehicle {
+    String color;
+    Car( int year, int speed, int cost,String coordinates,String color) {
+        this.year = year;
+        this.speed = speed;
+        this.cost = cost;
+        this.color = color;
+        this.coordinates = coordinates;
+    }
+
+    public void print() {
+        System.out.println(year+" "+speed+" "+cost+" "+coordinates+" "+color);
```

Comment:
You could print info more understandable:

 String str = String.format(
            "Car information"
                + " Year: %1$s, Speed : %2$s, Cost%2$s....",
            year, speed, cost);
  
        // Print and display the formatted string
        System.out.println(str);